templates/shared_services/redcap/REDME.md

```diff
@@ -0,0 +1,14 @@
+# REDCap Shared service
+
+The REDCap shared service requires a docker image to run the webapp.
+Due to the [licencing conditions](https://projectredcap.org/partners/termsofuse/)
+this cannot be pushed to a public registry. Therefore, to deploy a REDCap shared service
```

Comment:
Would be good to push to private repository when [TRE-3](https://ucldata.atlassian.net/browse/TRE-3) done

Reply:
Ah, I misread as "copying zip file because we can't push image to a registry"